### PR TITLE
Improve instruction on how to run the sample

### PR DESCRIPTION
**Pull request by manhg**
* Add env variable to configure example app.
* Link to test Google OAuth

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,26 @@ This is a simple blogging engine with a few basic features:
...
 and bad design. It's a demonstration of the surface API of Authboss and how the library can be used to make a functioning web
 project, to use this sample as anything else is malpractice.
+
+## Get started
+
+1. Copy env_sample to .env and edit OAuth client id/secret
+
+2. Build app
+```bash
+go get github.com/go-authboss/authboss
```
Comment (edwinthinks) [inline README.md]: I think this should be `go get gopkg.in/authboss.v1`?

**Review by edwinthinks — CHANGES_REQUESTED** [review]
I left a minor change suggestion.

Currently the master branch has a login user hardcoded and registering new users do not work due to hardcoded password input. I'd specifically say that in the README until it has been modified.

Thanks for improving the documentation. I was having a little difficulty in understanding it on first look. 👍

**Review by cuongquay — APPROVED** [review]

**Comment by aarondl** [comment]
Hey there. Apologies for lateness, this project's been at the very bottom of the priority list (with Authboss just above it I'm afraid :( )

Can we get a file extension for `env_sample` as it's clearly a shell script.

Also I believe @cintosyntax is correcti n that the go get (until 2.0) should be `go get gopkg.in/authboss.v1`

**Comment by manhg** [comment]
Thanks for feedback.
Sorry I made an mistake caused it closed. I opened another with fixes here https://github.com/go-authboss/authboss-sample/pull/15

**Pull request closed without merging**